lib/project_and_version_value.rb

```diff
@@ -1,55 +1,53 @@
 module ProjectAndVersionValue
-
-  def project_baseline project_id, baseline_id
+  def project_baseline(project_id, baseline_id)
     baselines = {}
-    if Evmbaseline.exists?(:project_id => project_id)
+    if Evmbaseline.exists?(project_id: project_id)
       if baseline_id.nil?
-        baselines = Evmbaseline.where("project_id = ? ", project_id).order("created_on DESC").first.evmbaselineIssues
+        baselines = Evmbaseline.where('project_id = ? ', project_id).order('created_on DESC').first.evmbaselineIssues
       else
-        baselines = Evmbaseline.where("id = ? ", baseline_id).first.evmbaselineIssues
+        baselines = Evmbaseline.where('id = ? ', baseline_id).first.evmbaselineIssues
       end
     end
     baselines
   end
 
-  def project_issues proj
-    issues = Issue.cross_project_scope(proj, "descendants").
-              where( "start_date IS NOT NULL AND due_date IS NOT NULL")
+  def project_issues(proj)
+    Issue.cross_project_scope(proj, 'descendants')
+      .where('start_date IS NOT NULL AND due_date IS NOT NULL')
   end
 
-  def project_costs proj
-    costs = Issue.cross_project_scope(proj, "descendants").
-              select("MAX(spent_on) AS spent_on, SUM(hours) AS sum_hours").
-              where( "start_date IS NOT NULL AND due_date IS NOT NULL" ).
-              joins(:time_entries).
-              group("spent_on").collect { |issue| [issue.spent_on, issue.sum_hours] }
+  def project_costs(proj)
+    Issue.cross_project_scope(proj, 'descendants')
+      .select('MAX(spent_on) AS spent_on, SUM(hours) AS sum_hours')
+      .where('start_date IS NOT NULL AND due_date IS NOT NULL')
+      .joins(:time_entries)
+      .group('spent_on').collect { |issue| [issue.spent_on, issue.sum_hours] }
```

Comment:
Prefer `map` over `collect`.<br>Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.